src/app.js

```diff
@@ -1,21 +1,7 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const { moveFile } = require('./moveFile');
 
-module.exports = sum;
+const args = process.argv;
+
```

Comment:
Implement error handling here to check if the source or destination arguments are missing